server/src/test/java/org/elasticsearch/cluster/routing/allocation/decider/DiskThresholdDeciderUnitTests.java

```diff
@@ -110,6 +112,61 @@ public void testCanAllocateUsesMaxAvailableSpace() {
             "disk space than the maximum allowed [90.0%]"));
     }
 
+    public void testCannotAllocate() {
```

Comment:
can you make this method name more descriptive? What scenario is being tested here?

Reply:
++